spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Comment do
+  context "Validate" do
+    it 'should have a valid factory' do
+      FactoryGirl.create(:comment)
+    end
+
+    it "Comment must belongs to article" do
+      comment = FactoryGirl.create(:comment)
+
+      expect(comment.respond_to?(:article)).to be true
+    end
+
+    it "Should pass create comment" do
+      expect do
+        comment = FactoryGirl.create(:comment)
```

Comment:
@phendych oh ok, ini harusnya gak perlu soalnya ini ngetest Rails' punya create method, which is bukan code kita